src/utils/nimbledroid/index.jsx

```diff
@@ -0,0 +1,103 @@
+const percentageSymbol = (GV, WV, targetRatio) => ((GV < (targetRatio * WV)) ? '+' : '');
+
+const ratioWithTarget = (GV, WV, targetRatio) => GV / (targetRatio * WV);
+
+export const sortSitesByTargetRatio = (a, b) => {
+  const aRatio = a.GV / a.WV;
+  const bRatio = b.GV / b.WV;
+  let retVal;
```

Comment:
The rest of this function could be simplified to be just `return bRatio - aRatio` 😄